test/ClangImporter/MixedSource/submodule.swift

```diff
@@ -2,6 +2,7 @@
 
 @_exported import Mixed
 @_exported import Mixed.Submodule
+import func Mixed.Submodule.fromSubmodule
```

Comment:
Is this an addition to the language, or is it a bug fix? (Ask Doug if you aren’t sure.)

Does it preclude adding scoped nested imports later? There’s a lot of code in the compiler that seems oriented towards supporting that eventually.

Reply:
@brentdax Thanks for the review! This is a bug fix (sorry, I forgot to add that to the PR description), we currently already support scoped imports on a submodule, e.g:

https://github.com/apple/swift/blob/853c0f545b61f5165c7c49fb24e675f305bdc760/test/IDE/Inputs/FooOverlay.swift#L3

We just didn't correctly handle the case where you're doing it for a submodule of the Clang part of a mixed source project.

I don't *think* this precludes adding scoped nested imports, though it'll probably be a bit tricky to implement if we continue to support scoped imports on submodules using this syntax. Ideally we'd require you to use some form of module qualification such as `Module.SubModule::Type`.